pkgs/development/python-modules/cryptolyzer/default.nix

```diff
@@ -30,6 +31,14 @@ buildPythonPackage rec {
     hash = "sha256-rRiRaXONLMNirKsK+QZWMSvaGeSLrHN9BpM8dhxoaxY=";
   };
 
+  patches = [
+    # https://gitlab.com/coroner/cryptolyzer/-/merge_requests/4
+    (fetchpatch {
+      url = "https://gitlab.com/coroner/cryptolyzer/-/commit/5ba89bc0bcc23fc7e8cd3eefb2d78b5a63f4fc66.patch";
+      hash = "sha256-G6ToCuWKSitDjmWQ6f4UqgnYItEUtFlsAbZPyxeRr8E=";
+    })
```

Comment:
Until the MR is merged and there is a commit that definitely exists in the upstream history, please vendor the patch as a file.

See the sections "Fetching patches" and "Vendoring patches" in: [`/pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/58f481134c4cb609305834818811341dfdb923cd/pkgs/README.md)